packages/svelte-cloudinary/tests/GetCldOgImageUrl/GetCldOgImageUrl.test.ts

```diff
@@ -0,0 +1,47 @@
+import { render, screen, cleanup } from '@testing-library/svelte';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+import { CldOgImage } from '$src/index';
+import { tick } from 'svelte';
+
+describe('CldOgImage', () => {
+	afterEach(() => {
+		cleanup();
+	});
+
+	it('should render a Cloudinary image with the given attributes', () => {
+		render(CldOgImage, {
```

Comment:
so for these tests, we want to test the [`getCldOgImageUrl` helper](https://svelte.cloudinary.dev/helpers/getcldogimageurl), rather than the component. An example test could be checking that the `url` string it returns contains the default width an height. You can do that something like this:

```ts
expect(url).toContain('w_1200');
expect(url).toContain('h_627');
```

I think we should start with tests for the following:

- Check that [the defaults](https://github.com/cloudinary-community/svelte-cloudinary/blob/main/packages/svelte-cloudinary/src/helpers/getCldOgImageUrl.ts#L34-L36) are set correctly (partial example above)
- Check that overwriting the defaults works (pass in a custom width, height, and format then check that's being used in the url)
- Check that the environment variable config is being respected ([example](https://github.com/cloudinary-community/svelte-cloudinary/blob/main/packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts#L47-L64))

You can remove the testing-library svelte stuff from this file, since the helper will work without it! Does this make sense?

Reply:
oh, and you can safely remove the turtle image too